models/tornado_cash/tornado_cash_deposits.sql

```diff
@@ -35,13 +35,13 @@ FROM
                 AND et.block_time >= (select min(evt_block_time) from {{ source('tornado_cash_ethereum','eth_evt_Deposit') }})
```

Comment:
Can't comment on the actual line but line 32  `source('ethereum','transactions_0006'`) should be ` source('ethereum','transactions')`.

Reply:
i believe that source was used specifically for performance reasons, but agreed to revert back if we're comfortable with new performance enhancement changes (could then also remove from source file)

Reply:
@soispoke can you update this?

Reply:
@jeff-dude Done